packages/react/src/reactrouterv6.tsx

```diff
@@ -179,28 +170,28 @@ export function withSentryReactRouterV6Routing<P extends Record<string, any>, R
     return Routes;
   }
 
-  let isBaseLocation: boolean = false;
-  let routes: RouteObject[];
+  let previousLocation: Location | null = null;
```

Comment:
l: As discussed we can change this to a boolean

Reply:
[af715ba](https://github.com/getsentry/sentry-javascript/pull/6609/commits/af715ba0f8b1fada65af3961a56e4216cb0e1e42)